spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -9,29 +9,44 @@
   end
 
   describe '#destroy' do
-    let(:idv_session) { double }
-    let(:flow_session) { {} }
-    let(:pii) { { first_name: 'Jane' } }
-
     before do
       allow(idv_session).to receive(:clear)
       allow(subject).to receive(:idv_session).and_return(idv_session)
-      controller.user_session['idv/doc_auth'] = flow_session
-      controller.user_session['idv/in_person'] = flow_session
+      controller.user_session['idv/doc_auth'] = { idv_doc_auth_session: true }
+      controller.user_session['idv/in_person'] = { idv_in_person_session: true }
+      controller.user_session['idv/inherited_proofing'] =
+        { idv_idv_inherited_proofing_session: true }
       controller.user_session[:decrypted_pii] = pii
     end
 
-    it 'deletes idv session' do
-      expect(idv_session).to receive(:clear)
+    let(:idv_session) { double }
+    let(:flow_session) { {} }
```

Comment:
Related to my previous comment, as it stands currently, this is unused. (But I'd expect this to be resolved with the suggestion in the other comment)

Reply:
If I am following you, yes; I'd have to add _something_ to `flow_session` to not get a false-positive.